README.md

```diff
@@ -1,45 +1,99 @@
+# Sherlock Platform
+
+## About
+
+This is the customized IntelliJ platform for Sherlock Project - an IntelliJ-based implementation of the new GPU Profiler tool. This is based
+off the IntelliJ Platform Version 2.1. To read more on the IntelliJ Platform documentation scroll down to the IntelliJ IDEA Community
+Edition section.
+
+## Developer Documentation
+
+### Copyright for Source Code Headers
+
+Every file containing source code must include copyright and license information.
+Make sure you add the following header when creating a new source file:
```

Comment:
Since this is a public repo forked from Jetbrains, I am not sure if we can do this.  For now, I think we should remove this copyright section to be on the safe side.